buku

```diff
@@ -595,31 +607,31 @@ class BukuDb:
             Must start and end with comma. Default is None.
         desc : str, optional
             Description of the bookmark. Default is None.
-        immutable : int, optional
+        immutable : int
             Indicates whether to disable title fetch from web.
-            Default is 0.
-        delay_commit : bool, optional
+            Default is 0 ("no"); the "yes" value is 1.
+        delay_commit : bool
             True if record should not be committed to the DB,
             leaving commit responsibility to caller. Default is False.
-        fetch : bool, optional
+        fetch : bool
             Fetch page from web and parse for data
 
         Returns
         -------
         int
-            DB index of new bookmark on success, -1 on failure.
+            DB index of new bookmark on success, None on failure.
```

Comment:
Same here. I prefer to keep the integral values integral. Also, this breaks the caller side for third-party utilities using this API.